Gemfile

```diff
@@ -13,7 +13,6 @@ gem 'aws-sdk-ses', '~> 1.6'
 gem 'aws-sdk-sns'
 gem 'barby', '~> 0.6.8'
 gem 'base32-crockford'
-gem 'blueprinter', '~> 0.25.3'
```

Comment:
Confused how this diff is here since it was also in https://github.com/18F/identity-idp/pull/7376 🤔

Reply:
I. dont know. i noticed that as well.